medicai/models/unetr/unetr.py

```
@@ -2,118 +2,142 @@
-        ]
+        *image_size, _ = input_shape
+        feat_size = tuple(img_d // encoder.patch_size for img_d in image_size)
+
+        # Get intermediate vectores
+        pyramid_outputs = encoder.pyramid_outputs
+        required_keys = {"P5", "P8", "P11"}
+        if not required_keys.issubset(pyramid_outputs.keys()):
+            raise ValueError(
+                f"The encoder's `pyramid_outputs` is missing one or more required keys. "
+                f"Required: {required_keys}, Available: {set(pyramid_outputs.keys())}"
+            )
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)